can-compute-stream.js

```diff
@@ -1,80 +1,163 @@
 var compute = require('can-compute');
-var define = require('can-define');
 var Kefir = require('kefir');
 var makeArray = require("can-util/js/make-array/make-array");
 var isArray = require("can-util/js/is-array/is-array");
 var assign = require("can-util/js/assign/assign");
+var define = require('can-define');
```

Comment:
Why would can-define be needed?  You might want to add jshint (you can see how other projects have it).